common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -117,6 +117,12 @@ class ProctoringFields(object):
         scope=Scope.settings,
     )
 
+    def _get_course(self):
+        """
+        Return course by course id.
+        """
+        return self.descriptor.runtime.modulestore.get_course(self.course_id)  # pylint: disable=no-member
```

Comment:
Is this where we will soon be using course blocks api?  I imagine this is our only choice until the performance issue on that gets worked out.  Do we need a separate story around that, or do we wait until @nasthagiri's team gets to it?